modules/integration/src/test/scala/scala/cli/integration/RunPipedSourcesTestDefinitions.scala

```diff
@@ -202,4 +202,14 @@ trait RunPipedSourcesTestDefinitions { _: RunTestDefinitions =>
       expect(output == expectedOutput)
     }
   }
+
+  test("running script snipped should not create workspace dir in cwd") {
+    emptyInputs.fromRoot { root =>
+      os.proc(TestUtil.cli, "-e", """println("Hello")""", extraOptions)
+        .call(cwd = root)
+        .out.trim()
```

Comment:
using the right quotation marks is necessary when passing snippets on Windows
```suggestion
      val quotation = TestUtil.argQuotationMark
      val msg = "Hello"
      val res = os.proc(TestUtil.cli, "-e", s"""println($quotation$msg$quotation")""", extraOptions)        
        .call(cwd = root)
        .out.trim()
      expect(res == msg)
```